content/tutorials/manage-the-rippled-server/configuration/run-rippled-as-a-validator.md

```diff
@@ -1,74 +1,191 @@
 
-5. Configure the validator host machine's firewall to only accept inbound connections from its public-facing rippled(s).
+1. [Enable validation](#enable-validation-on-your-rippled-server) on your `rippled` server.
 
-Remember to restart `rippled` for config changes to take effect.
+2. Set up stock `rippled` servers. For more information, see [Install rippled](install-rippled.html).
 
-Take care not to publish the IP address of your validator.
+3. Configure your validator and stock `rippled` servers to run in a [cluster](cluster-rippled-servers.html).
 
+4. In your validator's `rippled.cfg` file:
 
-## Domain Verification
+    1. Copy the `[ips_fixed]` IP address list you defined in the previous cluster configuration step and paste it under `[ips]`.
 
-Network participants are unlikely to trust validators without knowing who is operating them. To address this concern, validator operators can associate their validator with a web domain that they control.
+        The `[ips_fixed]` and `[ips]` values must be identical and contain only the IP addresses and ports of the stock `rippled` servers you clustered with your validator. This ensures that your validator connects to only the stock `rippled` servers that you control.
 
-1. Find your validator public key by running the following on the validator server:
+    2. Set `[peer_private]` to `1` to prevent your validator's IP address from being forwarded. For more information, see [Private Peers](peer-protocol.html#private-peers).
 
+        **Warning:** Be sure that you don't publish your validator's IP address via any other channels as well.
+
+5. Configure your validator host machine's firewall to accept inbound connections from only the stock `rippled` servers you clustered it with.
+
+6. Restart `rippled`.
+
+        $ sudo systemctl restart rippled.service
+
+***TODO: should we provide some tips on how to test that the firewall has been configured correctly? maybe ping the validator IP address from a machine outside the cluster? The test that the cluster is set up correctly (peers command) is covered in the clustering doc.***
+
+
+
+## Request Domain Verification
+
+***TODO: I'm unclear on what constitutes domain verification. Is it simply associating the validator with a web domain - like putting your public key on a website and claiming ownership? Or is it additionally getting your validator listed on XRP Charts or some other site like https://bithomp.com/validators or https://peerisland.com/validators?***
```

Comment:
Domain verification is associating your validator with a domain name. Right now, that can only happen by going through us and having us "check your work" essentially.